### ISSUE TITLE
Show rusting schedule of trains and which phase they cause

### ISSUE THREAD

**Issue by jenf** [tobymao/18xx#258]
Extend the current trains tab to add

Rusting schedule
Phase
Number of ORs

Also add the certificate limits for the current player count.

**Comment by dfannius** [comment]
Also the window for buying privates.

In general I think it would be good to have a free-form text field for phase info (in addition to anything else) because any individual game might have information that's not easily automatable. But maybe if we don't need it yet for 1889 it can wait.

**Comment by sparr** [comment]
@jenf if there was to be a display of all of the phase info (trigger train, train rusting, tile colors, offboard colors, number of ORs, etc), would the train tab still be the appropriate place to put it?

**Comment by jenf** [comment]
Maybe it's not 'trains' any more, but it's often on the same table in physical games

**Comment by dfannius** [comment]
Yeah, my long-term plan for the Trains tab was for it to become a Phases tab with all of the above information.
